files/image_config/interfaces/interfaces-config.sh

```diff
@@ -18,6 +18,8 @@ fi
 
 [ -f /var/run/dhclient.eth0.pid ] && kill `cat /var/run/dhclient.eth0.pid` && rm -f /var/run/dhclient.eth0.pid
 
+ip addr flush dev lo
```

Comment:
Does it make more sense to move this to the end of the file and change the sequence to:

```
ifdown --force lo
ip addr flush dev lo
ifup lo
```

Reply:
Yes,  it grouped better that way, and verified to be working.  Code changed.